.vscode/settings.json

```diff
@@ -34,8 +34,8 @@
   ],
   "pylint.importStrategy": "fromEnvironment",
   "pylint.args": [
-    "--rcfile=.pylintrc"
+    "--rcfile=./examples/python/.pylintrc"
   ],
-  "python.analysis.typeCheckingMode": "standard",
+  "python.analysis.typeCheckingMode": "basic",
```

Comment:
why and what's the difference?

Reply:
My initial thinking is that "basic" is more lenient in terms of reporting static type warnings/errors and might make it easier for others to contribute. But on a second thought, we should probably still make it standard to ensure code quality -- I will revert the change